index.html

```diff
@@ -187,6 +187,39 @@ <h2>Skills</h2>
                     </ul>
                 </section>
             </section>
+            <section class="form-container">
+                <div class="form-title">
+                    <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in</h3>
+                </div>
+                <form action="https://formspree.io/f/xayajwko" method="POST">
```

Comment:
According to the network tab in chrome web tools, it looks like the form info it's not ended in your e-mail, I kindly invite you to check this, I just fill the form sometimes just to check it. If you receive the info (name, email,  text) ignore this, if you don't receive it, please work on it. 🥇

Reply:
I do receive the info in my email.....Is that what you mean?

Reply:
Yes, if you receive it without any problem please continue to the next step. 😄